native/cocos/scene/RenderScene.cpp

```diff
@@ -109,6 +109,7 @@ void RenderScene::removeCamera(Camera *camera) {
 void RenderScene::removeCameras() {
     for (const auto &camera : _cameras) {
         camera->detachFromScene();
+        CC_SAFE_DESTROY(camera);
```

Comment:
just invoke `camera->destroy()` as `camera->detachFromScene()` is invoked above.

Reply:
Where is camera->detachFromScene() called？

Reply:
1